src/components/modules/credentials/ui/ShareCredentialModal.tsx

```
@@ -68,6 +82,44 @@ export default function ShareCredentialModal({
           ))}
         </div>
 
+        <div className="mt-4 space-y-2">
+          <div className="text-xs text-zinc-300">ZK Predicate</div>
+          <div className="flex items-center gap-2">
+            <select
+              value={predicate.kind}
+              onChange={(e) =>
+                setPredicate({ kind: e.target.value as any, value: predicate.value })
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)